### ISSUE TITLE
Latest revisions of 4.1.0 causes my tablet's webview to crash

### ISSUE THREAD

**Issue by felipecrs** [dermotduffy/frigate-hass-card#1000]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Release with the issue:**

release-4.1.0

**Last working release (if known):**

**Browser and Operating System:**
Android 10, recent versions of Chrome, Firefox or Fully Kiosk (which uses Webview).

**Description of problem:**

I was able to pin point the issue to the Frigate card as mentioned here:

- https://github.com/home-assistant/frontend/issues/15781

Basically my HA dashboard loads in an unresponsive state very often. This was not happening few weeks ago. Most likely it was caused by some recent revisions of the 4.1.0 branch.

Please check the linked issue above for more details and also for a video demonstrating the problem.

**Javascript errors shown in the web inspector (if applicable):**

There are no Javascript errors.

**Additional information:**

**Comment by felipecrs** [comment]
Just to mention, I can easily reproduce the issue here. It happens very often.

**Comment by dermotduffy** [comment]
Huh. What do you do to reproduce it? Do you have a smallest config that creates this problem for you? Do you know roughly when you started to have this issue? (Sorry for all the questions, hoping for some way to pin this down!).

**Comment by felipecrs** [comment]
I just load my HA dashboard or reload it. Nothing special.

I will try to revert some commits and check how it works. I'll have to also revert my config, but that's ok for now.

**Comment by felipecrs** [comment]
I did not yet try older revisions, but some more news are:

1. I was able to reproduce the issue in my mobile phone as well
2. I was able to reproduce the issue on my laptop as well (but not yet on my desktop, maybe because it's very powerful)

Checking Chrome's task manager, I figured out that the CPU usage was very high constantly, even though the page is completely unresponsive and the Frigate card did not even show up:

![image](https://user-images.githubusercontent.com/29582865/224494244-281586e5-2de1-438c-b443-2adf31003fa2.png)

**Comment by felipecrs** [comment]
Ok, found it. The culprit is:

- https://github.com/dermotduffy/frigate-hass-card/pull/988

Reverting it alone on top of the current rlease-4.1.0 fixes the issue for me.

That's odd, because I suggested it in the first place, and because reverting causes the translations not to work for me. Maybe the hass object isn't always set at such early time and it causes this issue?

**Comment by felipecrs** [comment]
> @felipecrs Thanks for tracking it down! Honestly though, all that code looks so innocent. The language load code is not called when the `hass` object has not yet loaded. Does this happen when your language is set to English also? Is this only on Android? 

I did not try setting my language to English.

It's not only Android, I was able to reproduce on Windows too. The browser also makes no difference, as I was also able to reproduce on Firefox.

However, what I noticed is that it's more tied to happen on lower end devices.

**Comment by felipecrs** [comment]
Maybe it's because I have 3 cards at the same time as well?

**Comment by felipecrs** [comment]
Maybe it's because I have 3 cards at the same time as well?

**Comment by dermotduffy** [comment]
Theory: It's actually related to #982 which reworked the initialization logic, and somehow getting stuck in infinite init loop.

**Comment by felipecrs** [comment]
This was one of the PRs I tested first (I tried to git reset the git history one by one), and it ended up not being the culprit.

**Comment by felipecrs** [comment]
Just to summarize, I found out that:

1. This issue apparently only happens when I try to access HA from within my local network
2. Reverting https://github.com/dermotduffy/frigate-hass-card/pull/988 solves my issue: `git revert 7eb130ea54987c9ad36f20f10006482571d1cf9a`

**Comment by felipecrs** [comment]
As [suggested](https://github.com/dermotduffy/frigate-hass-card/issues/1010#issuecomment-1476774623) by @badnbusy, the following also works:

3. Set performance profile from low to high

**Comment by dermotduffy** [comment]
Sadly I still cannot reproduce this (incl. the performance profile), but I have a good idea where the issue might be -- just working on something else right now, then I'll come back to this and play around...
